### DISCUSSION TITLE
Consulta TPI (Agenda)

### DISCUSSION THREAD

**Discussion by matiaspalleres** [INGCOM-UNRN-P1/cursada-2023#229] [Preguntas y respuestas] [answered]
Estamos teniendo un problema , si una variable espera un entero y se ingresa una cadena provoca un bucle infinito, como podriamos atajar ese error o no lo tomamos en cuenta?

**Comment by martinvilu** [comment, marked as answer]
Hace que cargue una cadena y procesa desde ahi, cosa de rechazar lo que no sea válido.
